src/ASM/DomainDecomposition.C

```diff
@@ -305,9 +305,9 @@ std::vector<std::set<int>> DomainDecomposition::getSubdomains(int nx, int ny, in
             if (type == (basis == 1 ? 'D' : 'P'+basis-2) || (type == ' ' && basis < 2)) {
               IntVec eqns;
               getSAM()->getNodeEqns(eqns, node);
-              for (auto& it : eqns) {
-                if (it > 0) {
-                  auto geq = blocks[block+1].G2LEQ.find(it);
+              for (auto& eqn : eqns) {
```

Comment:
Here also, `for (int eqn : eqns)` isn't better?

Reply:
sure, i just did minimal changes.